src/main/webapp/i18n/en/iris.json

```diff
@@ -32,6 +32,19 @@
                     },
                     "hestiaSettings": "Hestia Settings",
                     "competencyGenerationSettings": "Competency Generation Settings",
+                    "proactivityBuildFailedEventEnabled": {
+                        "label": "Monitor submission build failures",
+                        "tooltip": "When enabled, Iris will proactively send help messages if a submission build fails."
+                    },
+                    "proactivityProgressStalledEventEnabled": {
+                        "label": "Monitor exercise performance progress",
+                        "tooltip": "When enabled, Iris will proactively send help messages if the student's exercise performance does not improve."
+                    },
+                    "proactivitySettings": {
+                        "title": "Proactivity Settings",
+                        "tooltip": "When any of options listed below are activated, Iris will proactively send help messages to students based on the selected conditions.",
+                        "parentDisabled": "Iris won't send proactive messages for this option because either the same option or the Iris is disabled in the parent settings."
```

Comment:
This sounds really clunky imo. Maybe rewrite it something along the lines of:

"Iris won't send proactive messages for this option because either option itself or Iris are disabled in the parent settings"

Not 100% sure though tbh what you are trying to say here. (Same for the german version)

Reply:
I have adjusted the translations according to your suggestion